### ISSUE TITLE
can not submit an new article anymore

### ISSUE THREAD

**Issue by rallek** [zikula/core#3157]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.4dev ciBuild 98
| PHP Version       | 5.5.30

### Expected behavior
after pressing publish in News the article should be saved without error message

### Actual behavior
after clicking at publish I do get the following error message:
````
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.

An exception occurred while executing 'DELETE FROM pagelock WHERE name = ? AND session = ?' with params ["Newsnews", "8b3b9814cf0249b454cc9b3ea20a7e93"]: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db17.pagelock' doesn't exist
````

The article is saved fine.

I doen't matter if short URL is enabled or not. This behavior was not during test with 1.4.1. News is the latest master.

**Comment by rallek** [comment]
stack trace:
````
[1] Doctrine\DBAL\Exception\TableNotFoundException: An exception occurred while executing 'DELETE FROM pagelock WHERE name = ? AND session = ?' with params ["Newsnews", "8b3b9814cf0249b454cc9b3ea20a7e93"]:

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db17.pagelock' doesn't exist
    at n/a
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 53

    at Doctrine\DBAL\Driver\AbstractMySQLDriver->convertException('An exception occurred while executing 'DELETE FROM pagelock WHERE name = ? AND session = ?' with params ["Newsnews", "8b3b9814cf0249b454cc9b3ea20a7e93"]:

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db17.pagelock' doesn't exist', object(PDOException))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 128

    at Doctrine\DBAL\DBALException::driverExceptionDuringQuery(object(Driver), object(PDOException), 'DELETE FROM pagelock WHERE name = ? AND session = ?', array('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php line 996

    at Doctrine\DBAL\Connection->executeUpdate('DELETE FROM pagelock WHERE name = ? AND session = ?', array('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93'), array('2', '2'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Exec/SingleTableDeleteUpdateExecutor.php line 55

    at Doctrine\ORM\Query\Exec\SingleTableDeleteUpdateExecutor->execute(object(Connection), array('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93'), array('2', '2'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php line 321

    at Doctrine\ORM\Query->_doExecute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 969

    at Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache(null, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 924

    at Doctrine\ORM\AbstractQuery->execute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Entity/Repository/PageLockRepository.php line 153

    at Zikula\PageLockModule\Entity\Repository\PageLockRepository->deleteByLockName('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Api/LockingApi.php line 273

    at Zikula\PageLockModule\Api\LockingApi->releaseLock('Newsnews', '')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Api/UserApi.php line 123

    at Zikula\PageLockModule\Api\UserApi->releaseLock(array('lockName' => 'Newsnews'))
        in  line 

    at call_user_func(array(object(UserApi), 'releaseLock'), array('lockName' => 'Newsnews'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1216

    at ModUtil::exec('PageLock', 'user', 'releaseLock', array('lockName' => 'Newsnews'), true, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1292

    at ModUtil::apiFunc('PageLock', 'user', 'releaseLock', array('lockName' => 'Newsnews'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/modules/News/lib/News/Controller/User.php line 300

    at News_Controller_User->create(null)
        in  line 

    at call_user_func_array(array(object(News_Controller_User), 'create'), array(null))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1214

    at ModUtil::exec('News', 'user', 'create', array(), false, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1266

    at ModUtil::func('News', 'user', 'create', array())
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php line 176

    at Zikula\Bundle\CoreBundle\Controller\LegacyController->getLegacyResponse('News', 'user', 'create', array(), false)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php line 90

    at Zikula\Bundle\CoreBundle\Controller\LegacyController->legacyAction(object(Zikula_Request_Http))
        in  line 

    at call_user_func_array(array(object(LegacyController), 'legacyAction'), array(object(Zikula_Request_Http)))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 144

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Zikula_Request_Http), '1')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 64

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php line 69

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 185

    at Symfony\Component\HttpKernel\Kernel->handle(object(Zikula_Request_Http))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/index.php line 23

[2] Doctrine\DBAL\Driver\PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db17.pagelock' doesn't exist
    at n/a
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php line 93

    at Doctrine\DBAL\Driver\PDOStatement->execute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php line 987

    at Doctrine\DBAL\Connection->executeUpdate('DELETE FROM pagelock WHERE name = ? AND session = ?', array('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93'), array('2', '2'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Exec/SingleTableDeleteUpdateExecutor.php line 55

    at Doctrine\ORM\Query\Exec\SingleTableDeleteUpdateExecutor->execute(object(Connection), array('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93'), array('2', '2'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php line 321

    at Doctrine\ORM\Query->_doExecute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 969

    at Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache(null, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 924

    at Doctrine\ORM\AbstractQuery->execute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Entity/Repository/PageLockRepository.php line 153

    at Zikula\PageLockModule\Entity\Repository\PageLockRepository->deleteByLockName('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Api/LockingApi.php line 273

    at Zikula\PageLockModule\Api\LockingApi->releaseLock('Newsnews', '')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Api/UserApi.php line 123

    at Zikula\PageLockModule\Api\UserApi->releaseLock(array('lockName' => 'Newsnews'))
        in  line 

    at call_user_func(array(object(UserApi), 'releaseLock'), array('lockName' => 'Newsnews'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1216

    at ModUtil::exec('PageLock', 'user', 'releaseLock', array('lockName' => 'Newsnews'), true, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1292

    at ModUtil::apiFunc('PageLock', 'user', 'releaseLock', array('lockName' => 'Newsnews'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/modules/News/lib/News/Controller/User.php line 300

    at News_Controller_User->create(null)
        in  line 

    at call_user_func_array(array(object(News_Controller_User), 'create'), array(null))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1214

    at ModUtil::exec('News', 'user', 'create', array(), false, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1266

    at ModUtil::func('News', 'user', 'create', array())
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php line 176

    at Zikula\Bundle\CoreBundle\Controller\LegacyController->getLegacyResponse('News', 'user', 'create', array(), false)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php line 90

    at Zikula\Bundle\CoreBundle\Controller\LegacyController->legacyAction(object(Zikula_Request_Http))
        in  line 

    at call_user_func_array(array(object(LegacyController), 'legacyAction'), array(object(Zikula_Request_Http)))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 144

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Zikula_Request_Http), '1')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 64

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php line 69

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 185

    at Symfony\Component\HttpKernel\Kernel->handle(object(Zikula_Request_Http))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/index.php line 23

[3] PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db17.pagelock' doesn't exist
    at n/a
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php line 91

    at PDOStatement->execute(null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php line 91

    at Doctrine\DBAL\Driver\PDOStatement->execute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php line 987

    at Doctrine\DBAL\Connection->executeUpdate('DELETE FROM pagelock WHERE name = ? AND session = ?', array('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93'), array('2', '2'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Exec/SingleTableDeleteUpdateExecutor.php line 55

    at Doctrine\ORM\Query\Exec\SingleTableDeleteUpdateExecutor->execute(object(Connection), array('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93'), array('2', '2'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php line 321

    at Doctrine\ORM\Query->_doExecute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 969

    at Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache(null, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 924

    at Doctrine\ORM\AbstractQuery->execute()
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Entity/Repository/PageLockRepository.php line 153

    at Zikula\PageLockModule\Entity\Repository\PageLockRepository->deleteByLockName('Newsnews', '8b3b9814cf0249b454cc9b3ea20a7e93')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Api/LockingApi.php line 273

    at Zikula\PageLockModule\Api\LockingApi->releaseLock('Newsnews', '')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/system/PageLockModule/Api/UserApi.php line 123

    at Zikula\PageLockModule\Api\UserApi->releaseLock(array('lockName' => 'Newsnews'))
        in  line 

    at call_user_func(array(object(UserApi), 'releaseLock'), array('lockName' => 'Newsnews'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1216

    at ModUtil::exec('PageLock', 'user', 'releaseLock', array('lockName' => 'Newsnews'), true, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1292

    at ModUtil::apiFunc('PageLock', 'user', 'releaseLock', array('lockName' => 'Newsnews'))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/modules/News/lib/News/Controller/User.php line 300

    at News_Controller_User->create(null)
        in  line 

    at call_user_func_array(array(object(News_Controller_User), 'create'), array(null))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1214

    at ModUtil::exec('News', 'user', 'create', array(), false, null)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/util/ModUtil.php line 1266

    at ModUtil::func('News', 'user', 'create', array())
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php line 176

    at Zikula\Bundle\CoreBundle\Controller\LegacyController->getLegacyResponse('News', 'user', 'create', array(), false)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php line 90

    at Zikula\Bundle\CoreBundle\Controller\LegacyController->legacyAction(object(Zikula_Request_Http))
        in  line 

    at call_user_func_array(array(object(LegacyController), 'legacyAction'), array(object(Zikula_Request_Http)))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 144

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Zikula_Request_Http), '1')
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 64

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php line 69

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 185

    at Symfony\Component\HttpKernel\Kernel->handle(object(Zikula_Request_Http))
        in /home/www/1.4.4.dev.koester123.de/Zikula_Core-1.4.4/index.php line 23
````

**Comment by rallek** [comment]
pagelock is installed and activated

**Comment by rallek** [comment]
Changing a news is throwing the same error.

Pages is working fine

**Comment by Guite** [comment]
This is fixed by #3158

**Comment by craigh** [comment]
The issue is IMO not fixed by #3158, but the *symptom* is fixed. The core (and News) should fully function without installing pagelock. A different solution is needed.

**Comment by Guite** [comment]
Closed in favour of #3162
